src.yaml

```diff
@@ -2,9 +2,6 @@ openapi: '3.0.0'
 
 info:
   title: Qase.io API
-  description:
```

Comment:
Some description should present
![image](https://user-images.githubusercontent.com/6592868/143848374-796de174-4ea2-4af3-be6c-5282edd8c541.png)

Reply:
please provide a simple "one-liner" instead of the markdown